Dockerfile_archive

```diff
@@ -36,6 +36,8 @@ RUN apt-get update --yes \
         rustc \
  && apt-get clean
 
+COPY ./scripts/ /usr/local/bin/
```

Comment:
Does this actually work, won't this create /usr/local/bin/scripts/gpg-* rather than /usr/local/bin/gpg-* ?

Reply:
Ah, nevermind. COPY does actually do the right thing here and doesn't work like ``cp -a``

Reply:
All good - yeah, its a little different to shell (./.github/workflows/python-package.yml)